www/app/locals/local/view.html

```diff
@@ -1,7 +1,7 @@
 <ion-view view-title="Ahora suena">
   <ion-header-bar align-title="left" class="bar-positive">
     <div class="buttons">
-      <button class="button ion-chevron-left" ui-sref="menu.locals"></button>
+      <button class="button ion-chevron-left" ui-sref="menu.locals" ng-click="vm.killPlayer(); vm.isPlaying = false"></button>
```

Comment:
Move the `vm.isPlaying = false` expression inside the `vm.killPlayer()` function